code-challenges.js

```diff
@@ -31,25 +52,61 @@ const temperature2 = 350
 const temperature3 = 212
 // Expected output: "212 is at boiling point"
 
+const boilingPoint = (num) => {
+    if(num === 42 ){
+        return `${num} is below boiling point`
+    } else if(num === 350){
```

Comment:
We want to make sure that functions are dynamic. This means that they will work for not just the variables provided in the problem. What happens if you passed in the number 82 into your function? A dynamic function should return "82 is below boiling point", but your function currently will not return anything.